docs/getting-started.md

```diff
@@ -10,49 +10,59 @@ The motivation for building this library was to address the performance limitati
 
 In a nutshell, the library provides:
- - A way to use a platform's native touch handling system for recognizing pinch, rotation and pan (besides a few other gestures).
- - The ability to define relations between gesture handlers, e.g. when you have a pan handler in `ScrollView` you can make that `ScrollView` wait until it knows pan won't recognize.
- - Mechanisms to use touchables that run in native thread and follow platform default behavior; e.g. in the event they are in a scrollable component, turning into pressed state is slightly delayed to prevent it from highlighting when you fling.
- - The possibility to implement smooth gesture interactions thanks to Animated Native Driver &mdash; interactions will be responsive even when the JS thread is overloaded.
 
+- A way to use a platform's native touch handling system for recognizing pinch, rotation and pan (besides a few other gestures).
+- The ability to define relations between gesture handlers, e.g. when you have a pan handler in `ScrollView` you can make that `ScrollView` wait until it knows pan won't recognize.
+- Mechanisms to use touchables that run in native thread and follow platform default behavior; e.g. in the event they are in a scrollable component, turning into pressed state is slightly delayed to prevent it from highlighting when you fling.
+- The possibility to implement smooth gesture interactions thanks to Animated Native Driver &mdash; interactions will be responsive even when the JS thread is overloaded.
 
 ## Installation
 
 Since the library uses native support for handling gestures, it requires an extended installation to the norm.
 
 ### With [React Native](http://facebook.github.io/react-native/) app (no Expo)
-#### Requirements
 
-| version | react-native version |
-| ------- | -------------------- |
-| 1.1.0+  | 0.57.2+              | 
-| <1.1.0  | 0.50.0+              |
+#### Requirements
 
-Note that if you wish to use [`React.createRef()`](https://reactjs.org/docs/refs-and-the-dom.html) support for  [interactions](interactions.md) you need to use v.16.3 of [React](https://reactjs.org/)
+| version | `react-native` version |
+| ------- | ---------------------- |
+| 1.1.0+  | 0.57.2+                |
+| <1.1.0  | 0.50.0+                |
 
+Note that if you wish to use [`React.createRef()`](https://reactjs.org/docs/refs-and-the-dom.html) support for [interactions](interactions.md) you need to use v16.3 of [React](https://reactjs.org/)
 
 #### JS
+
 First, install the library using `yarn`:
+
 ```bash
-  yarn add react-native-gesture-handler
+yarn add react-native-gesture-handler
 ```
 
 or with `npm` if you prefer:
+
 ```bash
-  npm install --save react-native-gesture-handler
+npm install --save react-native-gesture-handler
 ```
 
 #### Linking
+
+> **Important**: This step is only applicable for React Native 0.59 or lower. Since v0.60 linking happens automatically.
```

Comment:
I'd rephrase it to

```suggestion
> **Important**: You only need to do this step if you're using React Native 0.59 or lower. Since v0.60, linking happens automatically.
```